src/contextify.cc

```diff
@@ -180,170 +182,201 @@ class ContextWrap : public ObjectWrap {
         return true;
     }
 
-    static NAN_PROPERTY_GETTER(GlobalPropertyGetter) {
-        NanScope();
-        Local<Object> data = args.Data()->ToObject();
+    static void GlobalPropertyGetter(Local<String> property, const PropertyCallbackInfo<Value>& info) {
+        Local<Object> data = info.Data()->ToObject();
         ContextifyContext* ctx = ObjectWrap::Unwrap<ContextWrap>(data)->ctx;
-        if (!ctx)
-            NanReturnUndefined();
-        Local<Value> rv = NanNew(ctx->sandbox)->GetRealNamedProperty(property);
-        if (rv.IsEmpty()) {
-            rv = NanNew(ctx->proxyGlobal)->GetRealNamedProperty(property);
+
+        if (!ctx) {
+            info.GetReturnValue().SetUndefined();
+            return;
         }
-        NanReturnValue(rv);
+
+        Local<Value> rv = Nan::New(ctx->sandbox)->GetRealNamedProperty(property);
+
+//        if (rv.IsEmpty()) {
+//            rv = Nan::New(ctx->proxyGlobal)->GetRealNamedProperty(property);
+//        }
+
+        info.GetReturnValue().Set(rv);
     }
 
-    static NAN_PROPERTY_SETTER(GlobalPropertySetter) {
-        NanScope();
-        Local<Object> data = args.Data()->ToObject();
+    static void GlobalPropertySetter(Local<String> property, Local<Value> value, const PropertyCallbackInfo<Value>& info) {
```

Comment:
v8::Context::Scope is just a RAII wrapper around Enter() and Exit(), why not simply alias it under NAN for now, in case it changes in the future (like gaining an Isolate* param)?